src/utils/helpers/application/revalidateAll.ts

```diff
@@ -7,23 +7,36 @@ import {
 } from '../../types'
 import validate from '../../../formElementPlugins/defaultValidate'
 
-export const revalidateAll = async (
-  elementsState: ApplicationElementStates,
-  responsesByCode: ResponsesByCode,
-  currentUser: User,
+interface RevalidateAllProps {
+  elementsState: ApplicationElementStates
+  responsesByCode: ResponsesByCode
+  currentUser: User
+  sectionCode?: string
+  strict?: boolean
+}
```

Comment:
Changes here look good and useful. Just checking if you've thoroughly regression tested this method in its other places (Summary page access and Submit), so we don't get any unintended regressions slipping through. (Esp related to the removal of the `strict` flag)

Reply:
I have tested. It seemed ok, but I think it would be nice if you could check too.

I have reverted the change, so the `strict` check is back